pages/blueprintEdit/index.js

```diff
@@ -569,17 +563,24 @@ class EditBlueprintPage extends React.Component {
         {(inputComponents !== undefined && blueprint.components !== undefined && blueprint.packages !== undefined && (
           <div className="cmpsr-panel__body cmpsr-panel__body--sidebar">
             <div className="toolbar-pf">
-              <form className="toolbar-pf-actions">
-                <div className="form-group toolbar-pf-filter">
-                  <input
-                    type="text"
-                    className="form-control"
-                    id="cmpsr-blueprint-input-filter"
-                    aria-label={formatMessage(messages.filterByLabel)}
-                    placeholder={formatMessage(messages.filterByPlaceholder)}
-                    onKeyPress={(e) => this.getFilteredInputs(e)}
-                  />
-                </div>
+              <form className="toolbar-pf-actions pf-l-flex">
+                <TextInput
+                  className="pf-m-flex-1"
+                  type="text"
+                  id="cmpsr-blueprint-input-filter"
+                  aria-label={formatMessage(messages.filterByLabel)}
+                  placeholder={formatMessage(messages.filterByPlaceholder)}
+                  onKeyPress={(e) => this.getFilteredInputs(e)}
+                />
+                <Pagination
+                  className="pf-m-flex-2"
```

Comment:
There are a few issues with the layout as shown in the examples below. The input should fill the space available to it, but it gets really crunched. One of the reasons it gets crunched is the pagination control stays on the same row and wraps within it self. 

<img width="436" alt="Screen Shot 2020-07-10 at 5 13 35 PM" src="https://user-images.githubusercontent.com/21063328/87203597-f5bcd200-c2d0-11ea-903e-3bd311a8d1a2.png">
<img width="356" alt="Screen Shot 2020-07-10 at 5 13 16 PM" src="https://user-images.githubusercontent.com/21063328/87203600-f7869580-c2d0-11ea-818c-c1d91e68fc72.png">

Here are some suggestions for modifying this:

- A ratio of 1:2 isn't always accurate for the `<TextInput>`:`<Pagination>` controls. The pagination is more or less a fixed size (i.e. it doesn't grow or shrink when it's container grows). Whereas the input is elastic and can shrink or grow to any width. I'd suggest that the input and pagination should be more of a 1:1 radio. I'd also suggest that the input needs a minimum width that is ~50% of the container (i.e. ~equal in length to the pagination control), so that it isn't forced to be shorter than it should at smaller viewports.
- Looking at how the beta Toolbar component is built, they apply `flex-wrap: nowrap` to the pagination control. In the examples above, the input and pagination remain split on the same row, but the elements inside the pagination component start to wrap. Instead, it would be better to keep the pagination elements all in a single line, and move them down to the second row, and allow the input to take up the full width. Instead of adding the beta Toolbar component now, we can use the utility class to apply `nowrap` and also set this on a breakpoint so that at the smallest viewport sizes, it can start to wrap. The class for this is `pf-u-flex-nowrap-on-[breakpoint].

The combination of these changes results in the filter refusing to shrink any smaller when it's half the width, but also not making the pagination component wrap within itself to accommodate the width of the input, therefore forcing the whole pagination component to a new row. 

Here are some examples after making these changes:
<img width="603" alt="Screen Shot 2020-07-10 at 5 08 30 PM" src="https://user-images.githubusercontent.com/21063328/87204557-23a31600-c2d3-11ea-880f-20318b37db97.png">
<img width="369" alt="Screen Shot 2020-07-10 at 5 08 45 PM" src="https://user-images.githubusercontent.com/21063328/87203562-e0e03e80-c2d0-11ea-8c4a-4f1754073716.png">
<img width="278" alt="Screen Shot 2020-07-10 at 5 08 30 PM" src="https://user-images.githubusercontent.com/21063328/87203565-e3db2f00-c2d0-11ea-86a5-8c64c39fed0a.png">